src/lib.rs

```diff
@@ -46,6 +49,12 @@ pub const RENDER_QUANTUM_SIZE: usize = 128;
 /// Maximum number of channels for audio processing
 pub const MAX_CHANNELS: usize = 32;
 
+/// Global `Instant` to create `AudioTimestamp` returned by `AudioContext::getOutputTimestamp`
+/// on a timeline shared by all contexts. This can be usefull to tightly synchronise
+/// audio rendering with other processes (e.g. graphics)
+// @note: naming inspired by https://www.w3.org/TR/hr-time-3/#dfn-time-origin
+pub(crate) static TIME_ORIGIN: Lazy<Instant> = Lazy::new(Instant::now);
```

Comment:
Not completely sure this is ok to do that?

Reply:
maybe we could make it public too, I don't have any idea how this could be used in user space (cf. next comment)